config/accounts.go

```diff
@@ -0,0 +1,14 @@
+package config
+
+// Account represents a publisher account configuration
+type Account struct {
+	ID               string       `mapstructure:"id" json:"id"`
+	Disabled         bool         `mapstructure:"disabled" json:"disabled"`
+	PriceGranularity string       `mapstructure:"price_granularity" json:"price_granularity"`
```

Comment:
The account config structure was trying to match the corresponding Java account config model:[org/prebid/server/settings/model/Account.java](https://github.com/rubicon-project/prebid-server-java/blob/master/src/main/java/org/prebid/server/settings/model/Account.java)

I have removed GDPR settings since then, so I can remove everything else that is not relevant to the functionality added here.